docs/references/config/config.toml.md

```diff
@@ -0,0 +1,913 @@
+
+The default relative path translates to `$CMTHOME/config/addrbook.json`. In case `$CMTHOME` is unset, it defaults to
+`$HOME/.cometbft/config/addrbook.json`.
+
+### p2p.addr_book_strict
+Strict address routability rules disallow non-routable IP addresses in the address book. When `false`, private network
+IP addresses are enabled to be stored in the address book.
+```toml
+addr_book_strict = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+Set it to `false` for testing on private network. Most production nodes can keep it at `true`.
+
+### p2p.max_num_inbound_peers
+Maximum number of inbound peers.
+```toml
+max_num_inbound_peers = 40
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+The [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) and
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers) values together define how many P2P connections the node will
+maintain at maximum capacity, not including list of nodes in the [`p2p.persistent_peers`](#p2ppersistent_peers) setting.
+
+The connections are bidirectional, so any connection can send or receive blocks and other data. The separation into
+inbound and outbound setting only distinguishes the initial setup of the connection: outbound connections are initiated
+by the node while inbound connections are initiated by a remote party.
+
+Nodes on non-routable networks have to set their gateway to port-forward the P2P port for inbound connections to reach
+the node. Outbound connections can be initiated as long as the node has generic Internet access. (Using NAT or methods.)
+
+### p2p.max_num_outbound_peers
+Maximum number of outbound peers.
+```toml
+max_num_outbound_peers = 10
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+The [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) and
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers) values together define how many P2P connections the node will
+maintain at maximum capacity, not including list of nodes in the [`p2p.persistent_peers`](#p2ppersistent_peers) setting.
+
+The connections are bidirectional, so any connection can send or receive blocks and other data. The separation into
+inbound and outbound setting only distinguishes the initial setup of the connection: outbound connections are initiated
+by the node while inbound connections are initiated by a remote party.
+
+Nodes on non-routable networks have to set their gateway to port-forward the P2P port for inbound connections to reach
+the node. Outbound connections can be initiated as long as the node has generic Internet access. (Using NAT or methods.)
+
+### p2p.unconditional_peer_ids
+List of node IDs, to which a connection will be (re)established ignoring any existing limits.
```

Comment:
Not really like that.

We maintain at most `max_num_inbound_peers` at time. If a new connection arrives from a peer set as unconditional, we don't check whether this limited has been exceeded. Plus, unconditional peers does not count towards the number of inbound or outbound peers.

We do not connect to unconditional peers, this is for persistent peers. But we never reject a connection from them.

Reply:
Yes, that note is complete gibberish. It was copied from the original config.toml comment. Lemme rewrite it.
```suggestion
List of node IDs, that are allowed to connect to the node even when connection limits are exceeded.
```